### DISCUSSION TITLE
Changing opacity of markers

### DISCUSSION THREAD

**Discussion by russbate42** [lukas-reineke/indent-blankline.nvim#910] [General]
Hello folks!
I'm wondering if it is possible to change the opacity of the indentation markers?
I have a transparent theme and the markers show up as fairly thick/aggressive and it doesn't really match the aesthetic I'm going for.
It would be wonderful if there was something a little more subtle.

I look forward to your reply! 👍🏻

**Comment by lukas-reineke** [comment]
Sorry, transparent text is not supported in any terminal emulator I know, there is no way to do that.
You can try to change the color to something darker, and/or use a thinner character. There are examples in the help, `:help ibl.config.indent.char`
